src/bespokelabs/curator/llm/llm.py

```diff
@@ -37,79 +37,6 @@
 class LLM:
     """Interface for prompting LLMs."""
 
```

Comment:
I think that __init__ should be the first function in the class?

Reply:
Yes, absolutely! I was planning to mention this and move it, but for clarity thought i will remove the incorrect one and keep the correct one and then move it. Moved it now.